pkg/operator/status.go

```diff
@@ -280,6 +280,15 @@ func (optr *Operator) syncUpgradeableStatus(co *configv1.ClusterOperator) error
 		Reason: asExpectedReason,
 	}
 
+	configNode, err := optr.configClient.ConfigV1().Nodes().Get(context.Background(), ctrlcommon.ClusterNodeInstanceName, metav1.GetOptions{})
```

Comment:
the operator has a `nodeClusterLister`, could we read off that here instead of doing a get? we prefer not to do direct API calls for recurring syncs

Reply:
Ack, Addressed